autocomplete/definitions/namedTypes/mwseLogger/setLevel.lua

```diff
@@ -0,0 +1,13 @@
+return {
+	type = "method",
+	description = [[Set the log level. 
+You can pass in either a string representation of a logging level, or the corresponding numerical constant found in the `mwse.logLevel` table.
+The options are: `"TRACE"`, `"DEBUG"`, `"INFO"`, `"WARN"`, `"ERROR"` and `"NONE"`.
+
+This function does exactly the same thing as writing `log.level = newLogLevel`. 
+Use whichever one you prefer.
+]],
+	arguments = {
+		{ name = "newLogLevel", type = "mwseLogger.logLevel" }
```

Comment:
Since this also supports passing in string representations of log levels, which we don't codify using `---@alias`, could you add a `|string` to the argument type so LuaLS won't show warnings when using the method that way?